docs/API/plain_rules.md

```diff
@@ -106,28 +106,35 @@ from iotEvent
 where (cast(cast(bloodPressure?,String),double)>1.5 and type="BloodMeter")]
```

Comment:
I'm not paying too much attention to this file in this PR, deferring a more deep review in the final PR https://github.com/telefonicaid/perseo-fe/pull/592

NTC